.circleci/config.yml

```diff
@@ -41,20 +77,22 @@ jobs:
           name: check formatting
           command: |
             make format-server
-            git diff-index --quiet HEAD -- || (echo "Found unexpected changes!" && git diff && exit 1)
       - run:
+          parallelism: 4
           name: server tests
           command: |
-            pipenv run pytest -n 2 --cov=.
+            circleci tests glob server/tests/**.py | circleci tests split --split-by=timings | xargs pipenv run pytest -n auto
```

Comment:
what does this do? also we lost test coverage I believe, which is actually one of the more important things to run on CI cuz ppl (me) don't often run it locally

Reply:
CircleCI requires some special juju to actually run tests in parallel, you basically have to let it handle the chunking of tests to different nodes. 

Re: coverage I will have to see what I can do about that.